### DISCUSSION TITLE
My github education request has been rejected

### DISCUSSION THREAD

**Discussion by IwakiLeKiwi** [community/community#69040] [GitHub Education]
### Hi everyone,

I've applied several times for the github education student package but without success. I received several rejection messages like :

```
The most common verification issues and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.
```

<img src="https://dev.kanawa.fr/storage/images/request.png">

I've sent several photos showing my identity card and my college certificate.

I don't understand why all these refusals. What should I do? Can you help me with my request?

Thank you for your help.

**Comment by mantasu** [comment]
Hey, try [opening a ticket](https://support.github.com/request/education) (click **Contact us**) and the staff will get back to you (though may take some time).

**Reply by mantasu** [reply]
Also, check if it's not a translation issue. Here is an [answer](https://github.com/orgs/community/discussions/68558#discussioncomment-7163615) from a staff member.

**Comment by williamregal** [comment]
Exam dumps for the [Fortinet NSE4_FGT-7.2  exact questions answer](https://www.marks4sure.com/NSE4_FGT-7-2-exam.html) are collections of practice questions and solutions that closely resemble the format and content of the real test. They are extensively given by many websites and platforms on the internet. Candidates utilize our exam dumps as an additional study tool while they get ready for the NSE4_FGT-7.2 certification.

**Comment by IwakiLeKiwi** [comment]
It's ok
